__tests__/FSRS-5.test.ts

```diff
@@ -59,7 +59,7 @@ describe('FSRS-5', () => {
       scheduling_cards = f.repeat(card, now)
     }
     expect(ivl_history).toEqual([
-      0, 4, 14, 44, 125, 328, 0, 0, 7, 16, 34, 71, 142,
+      0, 4, 14, 44, 125, 328, 0, 0, 6, 14, 31, 65, 130,
```

Comment:
Why is it changed?

Reply:
> Why is it changed?

It was affected by `clipParameters`; it has now been reverted.